src/features/Details/components/HeroDetailsList/HeroDetailsList.tsx

```diff
@@ -0,0 +1,64 @@
+import { jestIdsMap } from '@utils/jestHelpers';
+import { useMemo } from 'react';
+import styles from './HeroDetailsList.module.scss';
+
+interface HeroDetailsListProps<T> {
+  details: T;
+  detailNamesMap: Record<string, string>;
+}
+
+export const HeroDetailsList = <T,>({
```

Comment:
**suggestion:** please remove the comma

Reply:
@krzysztofmendrekclearcode I don't think it's possible, because Typescript then "thinks" it's JSX code. Either way the comma was intentional